src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -178,6 +181,10 @@ class PillSelector extends React.Component<Props, State> {
             'show-error': !!error,
             'pill-selector-suggestions-enabled': suggestedPillsEnabled,
         });
+        const ariaAttrs = {
+            'aria-invalid': !!error,
+            'aria-errormessage': this.errorMessageID,
+        };
 
         return (
             <Tooltip isShown={!!error} text={error || ''} position="middle-right" theme="error">
```

Comment:
Looks like we are doing !!error in a few places, perhaps should make a new const hasError

Reply:
Done @priyajeet